packages/sdk/src/accountTransactions.ts

```diff
@@ -98,10 +143,37 @@ export class SimpleTransferWithMemoHandler
             amount,
         };
     }
+
+    toJSON(
+        transfer: SimpleTransferWithMemoPayload
+    ): SimpleTransferWithMemoPayloadJSON {
+        return {
+            toAddress: AccountAddress.toBase58(transfer.toAddress),
+            memo: transfer.memo.toJSON(),
+            amount: transfer.amount.microCcdAmount,
+        };
+    }
+
+    fromJSON(
+        json: SimpleTransferWithMemoPayloadJSON
+    ): SimpleTransferWithMemoPayload {
+        return {
+            toAddress: AccountAddress.fromBase58(json.toAddress),
+            // The first 2 bytes are the length of the memo buffer, so we need to remove them.
+            memo: new DataBlob(encodeHexString(json.memo.substring(4))),
```

Comment:
I think it would be suitable to define a helper function for this inverse of `DataBlob.toJSON` in the `types/DataBlob.ts` module to keep the transformation logic in one place. And of course use that where applicable in this file 😄